src/Code.gs

```diff
@@ -646,12 +549,117 @@ const ChatGPTApp = (function () {
     }
 
     if (verbose) {
-      console.log('Got response from openAI API');
+      console.log(`Got response from openAI API ${numberOfAPICalls}`);
```

Comment:
Adding this number without context is difficult to understand
Maybe we can remove it? (not so necessary now I think, we have capped the number of calls anywway)